lib/Field/FieldRepeatableControls.js

```diff
@@ -1,45 +1,47 @@
 import React from 'react';
-import { bool, number, string } from 'prop-types';
+import { bool, number } from 'prop-types';
 import cx from 'classnames';
 import { ButtonIcon } from '../ButtonNew/ButtonIcon/ButtonIcon';
 import { ButtonIconDanger } from '../ButtonNew/ButtonIcon/ButtonIconDanger';
 
 export function FieldRepeatableControls({
   repeatPosition,
   isLastRepeat,
-  isActive,
-  className
+  isActive
 }) {
   const label = `${repeatPosition < 9 ? '0' : ''}${repeatPosition + 1}`;
 
-  const classes = cx(
-    className,
-    'flex justify-end',
+  const buttonClasses = cx(
+    'mx-1',
     'transition ease-animation-curve duration-200',
     {
       'opacity-0 group-hover:opacity-100': !isActive
     }
   );
 
   return (
-    <div className={classes}>
-      {repeatPosition > 0 && <ButtonIconDanger name="trash" className="mx-1" />}
-      {repeatPosition > 0 && <ButtonIcon name="arrowUp" className="mx-1" />}
-      {!isLastRepeat && <ButtonIcon name="arrowDown" className="mx-1" />}
+    <div className="flex justify-end">
+      {repeatPosition > 0 && (
+        <ButtonIconDanger name="trash" className={buttonClasses} size="sm" />
```

Comment:
💚 We've attached the size information to the new buttons, so in this case you can do 
```suggestion
        <ButtonIconDanger name="trash" className={buttonClasses} size={ButtonIconDanger.sizes.sm} />
```

Not a blocking change, just a knowledge share! We thought it was a nice way of doing it, as most IDE's will automatically show you what sizes you have available through autocomplete, so you don't have to dip into GUI!

Reply:
Nice, thanks!
what do you think about limiting it in the button prop types too, so

```js
// ButtonNew/common.js
export const propTypes = {
  // ...
  // size: string, becomes
  size: oneOf(Object.keys(sizes)),
  // ...
}
```